File: API/API/Domain/Models/Villa.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace API.Domain.Models
{
    public class Villa
    {
        public int VillaId { get; set; }

        public string VillaName { get; set; }

        public string VillaPath { get; set; }

        public int NumberOfBeds { get; set; }

        public double SurfaceArea { get; set; }

        public double PriceOnline { get; set; }

        public double PriceOnSite { get; set; }

        [NotMapped] public string Description { get; set; }

        public IEnumerable<Reservation> Reservations { get; set; }

        public IEnumerable<VillaEquipment> Equipments { get; set; }

        public IEnumerable<VillaFormula> Formulas { get; set; }

        public IEnumerable<VillaLanguage> Languages { get; set; }

        public IEnumerable<VillaMedia> Medias { get; set; }
    }
}

File: HotelVillaApplication/app/src/main/java/be/cienfuegos/sandra/hotel_villa_application/api/model/Villa.java
package be.cienfuegos.sandra.hotel_villa_application.api.model;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class Villa implements Serializable {
    private int villaId;
    private String villaName;
    private String villaPath;
    private int numberOfBeds;
    private int surfaceArea;
    private double priceOnline;
    private double priceOnSite;
    private String description;
    private List<VillaMedia> medias;
    private List<Equipment> equipments;
    private List<Formula> formulas;
    private List<Date> bookedDates;
}
